sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1459,11 +1459,6 @@ class HiveDDLSuite
           sql(s"ALTER TABLE tbl UNSET TBLPROPERTIES ('${forbiddenPrefix}foo')")
         }
         assert(e2.getMessage.contains(forbiddenPrefix + "foo"))
-
-        val e3 = intercept[AnalysisException] {
-          sql(s"CREATE TABLE tbl (a INT) TBLPROPERTIES ('${forbiddenPrefix}foo'='anything')")
-        }
-        assert(e3.getMessage.contains(forbiddenPrefix + "foo"))
```

Comment:
The error message is 
```
"Table or view 'tbl' already exists in database 'default';"
```
This is soft of behavior change, as we check table existence first.

Reply:
Instead of deleting the test, we should change the name of the table